src/models/external-table.ts

```diff
@@ -32,6 +32,7 @@ export interface ComplexOptions {
   disabledColumns?: string[];
   computedColumns?: ComputedColumn[];
   rowSeparator?: boolean;
+  color? : boolean;
```

Comment:
@jdmarshall Could you change this name to 
shouldDisableColors?: boolean; 
instead?

Reply:
That doesn't seem to match up with the other arguments, but I'll try.